src/util/strencodings.cpp

```diff
@@ -589,7 +589,7 @@ bool ParseHDKeypath(const std::string& keypath_str, std::vector<uint32_t>& keypa
 
 void Downcase(std::string& str)
 {
-    std::transform(str.begin(), str.end(), str.begin(), [](unsigned char c){return ToLower(c);});
+    std::transform(str.begin(), str.end(), str.begin(), [](char c){return ToLower(c);});
```

Comment:
Just realizing that it's passing a silly identity function:
```c++
std::transform(str.begin(), str.end(), str.begin(), ToLower);
```
Anyhow, not going to file a PR for just that, I guess the compiler optimizes this away, maybe someone touching this code in the future can take it into account.